### ISSUE TITLE
Pass localization

### ISSUE THREAD

**Issue by Zorato** [eymengunay/php-passbook#26]
Hi! Are there any plans for adding pass localization support? Apple reference: https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/PassKit_PG/Chapters/Creating.html#//apple_ref/doc/uid/TP40012195-CH4-SW54

**Comment by swey** [comment]
Hi @Zorato,

I extended the lib and added support for localization. At the moment you can only get it in my fork's branch "feature/localization". It's basically tested. After some more testing I will merge it to master.
https://github.com/swey/php-passbook/tree/feature/localization

Usage:
```
		$localization = new Localization('de');
		$localization->addImage(  new Image('./public/logo-de.png', 'logo') );
		$localization->addStrings(array(
			'event_name'	=> 'Deutscher Name'
		));
		$pass->addLocalization( $localization );

		$localization = new Localization('en');
		$localization->addImage(  new Image('../public/logo-en.png', 'logo') );
		$localization->addStrings(array(
			'event_name'	=> 'English name',
		));
		$pass->addLocalization( $localization );
```
I hope @eymengunay will adopt my changes or make me a collaborator. Then I can push my feature branch directly to this repository.

**Comment by Zorato** [comment]
Hi, @swey.
Thanks for your work. I have extended @eymengunay library to fit my needs, but your code is more flexible and supports image localization. Hope it will be merged in eymengunay/php-passbook master branch.

**Comment by g-** [comment]
This has been merged in.

**Comment by g-** [comment]
Released in Version 1.2.0.
